src/bin/utils.ts

```diff
@@ -48,8 +48,30 @@ ${rule.sql}`,
 	writeAll(output, outputFile);
 };
 
+const getConfigModel = (
+	fileContents: Model | AbstractSqlModel | Config,
+	modelName?: string,
+): Model | AbstractSqlModel => {
+	if ('models' in fileContents) {
+		if (fileContents.models.length === 0) {
+			throw new Error('No models found in config file');
+		}
+		if (modelName != null) {
+			const model = fileContents.models.find((m) => m.modelName === modelName);
+			if (model == null) {
+				throw new Error(
+					`Could not find model with name '${modelName}', found: ${fileContents.models.map((m) => m.modelName).join(', ')}`,
+				);
+			}
+		}
+		return fileContents.models[0];
+	}
+	return fileContents;
+};
+
 export const getAbstractSqlModelFromFile = (
 	modelFile: string,
+	modelName: string | undefined,
```

Comment:
`modelName?: string | undefined`

Reply:
Apparently tests use this function which is why I didn't catch it (tests use a different tsconfig). I did intentionally make it required because for the cli use cases it should always be provided, but for tests it's a bit different - I think I'll make the tests bend to the better method for the real code